CI/before_script.msvc.sh

```diff
@@ -415,17 +413,23 @@ fi
 	if [ -z $APPVEYOR ]; then
 		cd $DEPS_INSTALL
 
+        # Boost's installer is still based on ms-dos API that doesn't support larger than 260 char path names
+        # We work around this by installing to root of the current working drive and then move it to our deps
+        
 		BOOST_SDK="$(real_pwd)/Boost"
+		CWD_DRIVE_ROOT=`powershell -command '(get-location).Drive.Root'`  # get the current working drive's root
 
 		if [ -d Boost ] && grep "BOOST_VERSION 106100" Boost/boost/version.hpp > /dev/null; then
 			printf "Exists. "
 		elif [ -z $SKIP_EXTRACT ]; then
 			rm -rf Boost
-			"${DEPS}/boost-1.61.0-msvc${MSVC_YEAR}-win${BITS}.exe" //dir="$(echo $BOOST_SDK | sed s,/,\\\\,g)" //verysilent
+            "${DEPS}/boost-1.67.0-msvc${MSVC_YEAR}-win${BITS}.exe" //DIR="${CWD_DRIVE_ROOT}Boost_temp" //VERYSILENT //NORESTART //SUPPRESSMSGBOXES //LOG="boost_install.log"			
+			mv "${CWD_DRIVE_ROOT}Boost_temp" ${BOOST_SDK}
```

Comment:
You might still want to have quotes around `${BOOST_SDK}`, don't want to break on paths with spaces in it.

Reply:
@ananace We can't support spaces whatever we do because Qt refuses to go anywhere with spaces.